src/Disks/DiskLocal.cpp

```diff
@@ -534,7 +534,7 @@ DiskLocal::DiskLocal(const String & name_, const String & path_, UInt64 keep_fre
     : IDisk(name_, config, config_prefix)
     , disk_path(path_)
     , keep_free_space_bytes(keep_free_space_bytes_)
-    , logger(getLogger("DiskLocal"))
+    , logger(getLogger(fmt::format("DiskLocal({})", name_)))
```

Comment:
If I understand this correctly, we will create a logger for every unique disk name if the logger doesn't exist yet. I don't think that this is desirable. Let's just use `DiskLocal` as logger name.